pkgs/development/coq-modules/coq-elpi/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, which, coq }:
+
+let params = {
+  "8.10" = {
+    version = "master";
+    rev = "bc7134deba1aacc7ecd2f5d1032bdf05b125c568";
+    sha256 = "188avk9irwjsbs5ya4ka01mpk3vw4397kv2rmsncqrrrsa1pdddk";
+  };
+};
+  param = params."${coq.coq-version}";
+in
+
+stdenv.mkDerivation rec {
+  name = "coq${coq.coq-version}-elpi-${param.version}";
```

not sure if this is more right, but I think this is "more correct"
```suggestion
  name = "coq-${coq.coq-version}-${coq.ocamlPackages.elpi.name}";
```
"coq8.10-elpi-master" as name to "coq-8.10-elpi-1.6.0"

however, since you're taking depending on a specific commit, the normal convention is
```suggestion
  pname = "coq-elpi";
  version = "unstable-2019-08-02";
```
I don't think i've seen a package that had dependencies as part of the title.